### DISCUSSION TITLE
Show name in developer tools states

### DISCUSSION THREAD

**Discussion by KTibow** [home-assistant/frontend#7581] [Other feature requests] [answered]
The developer tools states tab doesn't have a dedicated field for name. Some entities have cryptic entity IDs, and it'd be nice to filter by friendly_name. Yes, I know I can customize them, but I still think this would be a good FR (use case: changing ID would break automations/scripts/groups/UI).

**Comment by spacegaier** [comment]
You can already filter them via the "Attributes" column since the `friendly_name` is in there, but I agree having the name more prominently sounds good, especially since some users disable the attributes since they take too long to render in huge environments. 

My proposal is to add the name in the first column as I have done with the timestamp for the state change here: https://github.com/home-assistant/frontend/pull/7375

Something like this perhaps?
![image](https://user-images.githubusercontent.com/114137/97928702-e3771000-1d67-11eb-9be7-545c552faaf5.png)

**Comment by spacegaier** [comment, marked as answer]
Update: PR https://github.com/home-assistant/frontend/pull/7582 created

Update: PR was just merged.

**Reply by KTibow** [reply]
Remove the breaking change header before the bot applies the label! Quick!

**Reply by spacegaier** [reply]
What? Why? The bot is not looking at that in this repo AFAIK.

**Reply by KTibow** [reply]
Sorry, had a PR, and I couldn't remove the breaking-change label.
